theme/static/js/resourceLandingAjax.js

```diff
@@ -72,6 +72,41 @@ function shareable_ajax_submit(event) {
     return false;
 }
 
+function license_agreement_ajax_submit(event) {
+    // this sets if user will be required to agree to resource rights statement prior
+    // to any resource file or bag download
+    var form = $(this).closest("form");
+    var datastring = form.serialize();
+    var url = form.attr('action');
+    var element = $(this);
+    var action = $(this).closest("form").find("input[name='t']").val();
+    element.attr("disabled", true);
+
+    $.ajax({
+        type: "POST",
+        url: url,
+        dataType: 'html',
+        data: datastring,
+        success: function () {
+            element.attr("disabled", false);
+            if (action == "make_not_require_lic_agreement") {
+                element.closest("form").find("input[name='t']").val("make_require_lic_agreement");
+            }
+            else {
+                element.closest("form").find("input[name='t']").val("make_not_require_lic_agreement");
+            }
+            // page refresh is needed to update if license agreement popup to show or not prior
+            // to download of files/bag
+            window.location = window.location.href;
```

Comment:
@pkdash Not sure why it needs page refresh here. If it turns out this page refresh unavoidable, it would work better to scroll the page automatically to the license check location for better user experience rather than automatically scroll to the top of the page after refresh.